### PR TITLE
Add support for Journalpost#journalansvarlig

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

File: src/02 - Object types.md [hunk 1]
```diff
@@ -405,6 +405,8 @@ The following notation is used in the tables below:
...
+|      | string          | journalansvarlig              |    x     |      x      |           |  |  x   |  x   |   x   |  x   |               | <p>An identifier of the person responsible for this Journalpost (similar to M307 Saksmappe#saksbehandler)</p><p>&bull; may not be supported by all systems or partially supported</p>                                                                                                                  |                                                                                                                                                        |
+|      | string          | journalansvarligBrukerIdent   |    x     |      x      |           |  |  x   |  x   |   x   |  x   |               | <p>&bull; may not be supported by all systems or partially supported<                                                                                                                                                                                                                                  |                                                                                                                                                        |
```
Comment (teodoradobreva) [inline src/02 - Object types.md]: There is a partially closed tag at the end of the comment.

Comment (ivaylomitrev) [reply src/02 - Object types.md]: I checked this twice as these are the primary issues I get when editing the files with Atom and still got it wrong... :D

Good catch! Fixed.

File: src/02 - Object types.md [hunk 2]
```diff
@@ -405,6 +405,8 @@ The following notation is used in the tables below:
...
 | M015 | number          | journalpostnummer             |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                                                                                                                                                                                                                                        |                                                                                                                                                        |
+|      | string          | journalansvarlig              |    x     |      x      |           |  |  x   |  x   |   x   |  x   |               | <p>An identifier of the person responsible for this Journalpost (similar to M307 Saksmappe#saksbehandler)</p><p>&bull; may not be supported by all systems or partially supported</p>                                                                                                                  |                                                                                                                                                        |
```
Comment (douzounov) [inline src/02 - Object types.md]: `journalansvarlig` contains the name of the person, similarly to `saksansvarlig`. I think this is not reflected in the documentation correctly. We need the comment to state that `journalansvarlig` contains a person's name, while `journalansvarligBrukerIdent` contains the corresponding user ID.

Comment (ivaylomitrev) [reply src/02 - Object types.md]: Addressed. Please take a look at the change.

**Comment by ivaylomitrev** [comment]
Rebased on latest develop and squashed.

**Pull request merged**